### DISCUSSION TITLE
Welcome to tai Discussions!

### DISCUSSION THREAD

**Discussion by rupurt** [fremantle-capital/tai#182] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ihorkatkov** [comment]
@rupurt It's a great idea!

Hello everyone, my name is Ihor Katkov, and I love trading domain :D For the last three years, I'm working with Elixir, mostly as a professional nomad contractor. In my spare time, I'm doing algotrading, surfing/kitesurfing, and Elixir meetups for the Russian-speaking community.

@rupurt what do you think to have Slack/Discord (or channel in Elixir Slack) for the community?

Thanks

My Linkedin: https://www.linkedin.com/in/ihorkatkov/
My Telegram: @ikatkov

**Reply by rupurt** [reply]
Hey @ihorkatkov,

Speciba for the background :) 

Kite surfing sounds cool. I'd love to learn some time. 

I like the idea of a Slack/Discord but I can't commit to being present in them as this isn't my full time gig. I was hoping this async channel would open up a bit more of a passive community. If you'd like to create a Slack or Discord I'd be all for it.

**Comment by ukazap** [comment]
Hi @rupurt 

Out of curiosity, what's the meaning and the history behind the name 'Tai'?

Anyway, thanks for the project, looking forward to using it in prod and maybe contributing back someday.

**Reply by rupurt** [reply]
Howdy @ukazap,

It was originally going to be used for `Trading AI` + `tai` is a short name. Once I got deeper into the project I also liked the idea that this project was a component that's used to `tie` pieces of your trading system together.

> Anyway, thanks for the project, looking forward to using it in prod and maybe contributing back someday.

Cool. Lmk if you run into any issues getting up and going

**Reply by ukazap** [reply]
That's neat, thanks!!!
